tests/storage-buckets

```diff
@@ -22,6 +36,23 @@ if echo "${poolDriverList}" | grep -qwF "ceph"; then
   apt-get install --no-install-recommends --yes ceph-common
 fi
 
+# Clean up the build dir in case it hung around from a failed test.
+rm -rf /opt/minio
+mkdir -p /opt/minio
+
+# Download the minio server.
+curl -sSfL "https://dl.min.io/server/minio/release/linux-${arch}/minio" --output "/opt/minio/minio"
+chmod +x "opt/minio/minio"
+
+# Also grab the latest minio client to maintain compatibility with the server.
+curl -sSfL "https://dl.min.io/client/mc/release/linux-${arch}/mc" --output "/opt/minio/mc"
+chmod +x "opt/minio/mc"
+
+# Set the snap config key for minio and restart LXD to have it take effect.
+snap set lxd minio.path=/opt/minio
+snap restart lxd
```

Comment:
Lxd reload should be sufficient and snap restart has a bug with lxd so let's not use it here